Sources/MapboxMaps/Annotations/ViewAnnotationManager.swift

```diff
@@ -284,6 +292,90 @@ public final class ViewAnnotationManager {
         observers.removeValue(forKey: ObjectIdentifier(observer))
     }
 
+    // MARK: Framing
+
+    /// Calculates ``CameraOptions`` to fit the list of view annotations.
+    ///
+    /// - Important: This API isn't supported by Globe projection.
+    ///
+    /// - Parameter ids: The list of annotations ids to be framed.
+    /// - Parameter padding: See ``CameraOptions/padding``.
+    /// - Parameter bearing: See ``CameraOptions/bearing``.
+    /// - Parameter pitch: See ``CameraOptions/pitch``.
+    public func camera(for ids: [String], padding: UIEdgeInsets = .zero, bearing: CGFloat? = nil, pitch: CGFloat? = nil) -> CameraOptions? {
```

Comment:
better to name as `cameraForAnnotations`. this will align naming with other apis we have. wdyt?

Reply:
yes that's better